docs/_static/nuts-discovery.yaml

```diff
@@ -0,0 +1,115 @@
+openapi: "3.0.0"
+info:
+  title: Nuts discovery API spec
+  description: API specification for certificate related services hosted by the network authority
+  version: 0.1.0
+  license:
+    name: GPLv3
+paths:
+  /api/csr:
+    post:
+      operationId: submit
+      summary: "submit a PEM encoded CSR to the network authority"
+      tags:
+        - certificates
+      requestBody:
+        description: PEM encoded CSR
+        required: true
+        content:
+          text/plain:
+            schema:
+              type: string
+      responses:
```

Comment:
What if we return a identifier which uniquely identifies this submitted CSR?

- We can instruct vendors to communicate that ID to one of the operators using an authenticated channel (slack, for instance). CA operators can then accept that exact submitted CSR and sign a certificate.
- The vendor can retrieve the certificate using that ID later on (since the certificate is public anyway).

Reply:
This would be an extension to the process used in production. I'll create an issue for that: #51